auditeval/test.go

```diff
@@ -154,6 +154,7 @@ func getFlagValue(s, flag string) string {
 
 	var flagVal string
 	pttns := []string{
+		flag + `="(.*)"`,
```

Comment:
I'm going to accept this as-is, but maybe you and @yoavrotems could get together to discuss how to fix #35. I think we need regexes that will accept spaces before and after the quotes and this will apply to this pattern you've added as well as the next one.